xain/grpc/test_coordinator_logic.py

```diff
@@ -66,6 +66,9 @@ def test_start_training():
 
 
 def test_end_training():
+    # we need to participants so that we can check the status of the local update mid round
```

Comment:
s/to/two

Reply:
sorry I missed this suggestion the first time.